src/main/java/mate/academy/controller/BookController.java

```diff
@@ -32,4 +36,15 @@ public BookDto getBookById(@PathVariable Long id) {
     public BookDto createBook(@RequestBody CreateBookRequestDto bookDto) {
         return bookService.save(bookDto);
     }
+
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @DeleteMapping("/{id}")
+    public void delete(@PathVariable Long id) {
+        bookService.deleteById(id);
+    }
+
+    @PutMapping("/{id}")
+    public void updateBookById(@PathVariable Long id, @RequestBody CreateBookRequestDto bookDto) {
+        bookService.setById(id, bookDto);
```

Comment:
better not to confuse actions
```suggestion
        bookService.updateById(id, bookDto);
```